frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/employee/NewApplication/index.js

```
@@ -1,153 +1,1218 @@
+      ],
+
+      units: unit.map(unit => (
+        {
+          usageCategory: unit.usageType || "RESIDENTIAL",
+          usesCategoryMajor: unit.usageType || "RESIDENTIAL",
+          occupancyType: unit.usageFactor || "SELFOCCUPIED",
+          constructionDetail: {
+            builtUpArea: unit.area || "3000",
+            constructionType: unit.constructionType || null,
+          },
+          floorNo: parseInt(unit.floorNo) || 0,
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)